core/src/main/java/org/opensearch/sql/analysis/Analyzer.java

```diff
@@ -234,30 +243,28 @@ public LogicalPlan visitFilter(Filter node, AnalysisContext context) {
   }
 
   /**
-   * Ensure NESTED function is not used in GROUP BY, and HAVING clauses.
-   * Fallback to legacy engine. Can remove when support is added for NESTED function in WHERE,
-   * GROUP BY, ORDER BY, and HAVING clauses.
+   * Ensure NESTED function is not used in GROUP BY, and HAVING clauses. Fallback to legacy engine.
+   * Can remove when support is added for NESTED function in WHERE, GROUP BY, ORDER BY, and HAVING
+   * clauses.
+   *
    * @param condition : Filter condition
    */
   private void verifySupportsCondition(Expression condition) {
     if (condition instanceof FunctionExpression) {
-      if (((FunctionExpression) condition).getFunctionName().getFunctionName().equalsIgnoreCase(
-          BuiltinFunctionName.NESTED.name()
-      )) {
+      if (((FunctionExpression) condition)
+          .getFunctionName()
+          .getFunctionName()
+          .equalsIgnoreCase(BuiltinFunctionName.NESTED.name())) {
         throw new SyntaxCheckException(
             "Falling back to legacy engine. Nested function is not supported in WHERE,"
-                + " GROUP BY, and HAVING clauses."
-        );
+                + " GROUP BY, and HAVING clauses.");
       }
-      ((FunctionExpression)condition).getArguments().stream()
-          .forEach(e -> verifySupportsCondition(e)
-      );
+      ((FunctionExpression) condition)
+          .getArguments().stream().forEach(e -> verifySupportsCondition(e));
     }
   }
 
-  /**
-   * Build {@link LogicalRename}.
-   */
+  /** Build {@link LogicalRename}. */
```

Comment:
This fails checkstyle. Should checkstyle be disabled or can spotless be configured to allow this? There's also other checkstyle failures to lookout for.